lua/fishing_mod/cl_init.lua

```diff
@@ -65,60 +66,85 @@ hook.Add("InitPostEntity", "Init Fish Mod", function()
 	end
 
 end)
-
+local uiText = fishingmod.DefaultUIColors().uiText
+local bg = fishingmod.DefaultUIColors().uiBackground
+local crosshair = fishingmod.DefaultUIColors().crossHairColor
 hook.Add( "HUDPaint", "Fishingmod:HUDPaint", function()
 	local entity = LocalPlayer():GetEyeTrace().Entity
+	if fishingmod.ColorTable then 
+		crosshair = fishingmod.ColorTable.crossHairColor or fishingmod.DefaultUIColors().crossHairColor
+		uiText = fishingmod.ColorTable.uiText or fishingmod.DefaultUIColors().uiText
+		bg = fishingmod.ColorTable.uiBackground or fishingmod.DefaultUIColors().uiBackground
+	end
 	entity = IsValid(entity) and IsValid(entity:GetNWEntity("FMRedirect")) and entity:GetNWEntity("FMRedirect") or entity
-	if not IsValid(entity) then return end
-	
-	local xy = (entity:LocalToWorld(entity:OBBCenter())):ToScreen()
-	local sthx, sthy = 0, 0
-	xy.y = math.min(math.max(64, xy.y), ScrH() - 64)
-	local pad = 3
-		
-	if IsValid(entity) and (entity:GetPos() - LocalPlayer():GetShootPos()):Length() < 120 then
-		local data = fishingmod.InfoTable.Catch[entity:EntIndex()]
-		if(data and data.text) then
-			local text_ = string.Replace(string.Replace(data.text, "\t", ""), "  ", " ")
-			surface.SetFont("fixed_Height_Font")
-			surface.SetDrawColor(color_white.r, color_white.r, color_white.r, color_white.r)
-			sthx, sthy = surface.GetTextSize(data.text)
-			sthx, sthy = sthx + 8, sthy + 8
-			surface.SetDrawColor(0, 0, 0, 144)
-			surface.DrawRect(xy.x - sthx / 2 - pad, xy.y - sthy / 2 - 1 - pad, sthx + pad * 2, sthy + pad * 2)
-			surface.DrawRect(xy.x - sthx / 2 + 3 - pad, xy.y - sthy / 2 - 1 + 3 - pad, sthx - 6 + pad * 2, sthy - 6 + pad * 2)
-			draw.DrawText(string.Replace(text_, "\t", ""), "fixed_Height_Font", xy.x, xy.y - (sthy / 2), color_white, 1) -- \t key causes it to snap
-		end
+	if IsValid(entity) then
+		local xy = (entity:LocalToWorld(entity:OBBCenter())):ToScreen()
+		local sthx, sthy = 0, 0
+		xy.y = math.min(math.max(64, xy.y), ScrH() - 64)
+		local pad = 3
 			
-		data = fishingmod.InfoTable.Bait[entity:EntIndex()]
-		if(data and data.text) then
-			surface.SetFont("fixed_Height_Font")
-			surface.SetDrawColor(0, 0, 0, 144)
-			sthx, sthy = surface.GetTextSize(string.Replace(string.Replace(data.text,"\t",""),"\n",""))
-			sthx, sthy = sthx + 8, sthy + 8
-			surface.DrawRect(xy.x - sthx / 2 - pad, xy.y - sthy / 2 - 1 - pad + 16, sthx + pad * 2, sthy + pad * 1)
-			surface.DrawRect(xy.x - sthx / 2 + 3 - pad, xy.y - sthy / 2 + 2 - pad + 16, sthx - 6 + pad * 2, sthy - 6 + pad * 1)
-			draw.DrawText(string.Replace(data.text, "\t", ""), "fixed_Height_Font", xy.x, xy.y - (sthy / 2) + 15, color_white, 1)
+		if IsValid(entity) and (entity:GetPos() - LocalPlayer():GetShootPos()):Length() < 120 then
+			local data = fishingmod.InfoTable.Catch[entity:EntIndex()]
+			if(data and data.text) then
+				local text_ = string.Replace(string.Replace(data.text, "\t", ""), "  ", " ")

```

Comment:
same in this whole section variable names are not descriptive at all and questionable